src/DssSpell.t.base.sol

```diff
@@ -43,31 +43,6 @@ interface PsmAbstract {
     function buyGem(address usr, uint256 gemAmt) external;
```

Comment:
consider removing `PsmAbastract` and `AuthLike` as well and `dapp upgrade` dss-interfaces (along with dss-exec-lib once dss-interfaces have been upgraded as well) to import them from `dss-interfaces` (`PsmAbstract` and `WardsAbstract`) or wait till next `dss-exec-lib` release to cleanup those.
cc: @gbalabasquer